x/storage/keeper/keeper.go

```diff
@@ -1386,19 +1391,20 @@ func (k Keeper) UpdateGroupMember(ctx sdk.Context, operator sdk.AccAddress, grou
 			return err
 		}
 
-		groupMemberDetailEvent := &types.EventGroupMemberDetail{
-			Member:         opts.MembersToAdd[i],
-			ExpirationTime: types.MaxTimeStamp,
+		var memberExpiration *time.Time
```

Comment:
what is the motivation to change this?

Reply:
The `Expiration` changed from `time.Time` to `*time.Time`(to enable nil expiration).